src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,47 @@
+.alert {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: 9px 20px 9px 32px;
+  background: #004f71;
+  box-shadow: 0px 4px 20px rgba(91, 103, 112, 0.24);
+  position: fixed;
+  top: 72px;
+  width: calc(100% - 52px);
+
+  &--closed {
+    display: none;
+  }
+  &-container {
+    display: flex;
+    flex-direction: row;
```

Comment:
Can you add the following so the icon and text are better aligned vertically?
`margin: auto 0;`